app/book.py

```diff
@@ -0,0 +1,57 @@
+import json
+import xml.etree.ElementTree as ElTree
+
+
+class Book:
+    def __init__(self, title: str, content: str) -> None:
+        self.title = title
+        self.content = content
+
+    def display_console(self) -> None:
+        print(self.content)
+
+    def display_reverse(self) -> None:
+        print(self.content[::-1])
+
+    def display_book(self, display_type: str) -> None:
+        if display_type == "console":
+            self.display_console()
+        elif display_type == "reverse":
+            self.display_reverse()
+        else:
+            raise ValueError(f"Unknown display type: {display_type}")

```

Comment:
These functions violate Single responsibility principle. You must move them to separate classes